src/labels.js

```diff
@@ -69,10 +49,18 @@ class Labels {
     }
   }
 
-  static initMetadata (wallet) {
-    return Metadata.fromMasterHDNode(wallet._metadataHDNode, METADATA_TYPE_LABELS);
+  get readOnly () {
+    return false;
+  }
+
+  syncWallet () {
+    const syncWallet = () => {
+      this._syncWallet(() => Promise.resolve(), (e) => Promise.reject(e));
```

Comment:
This method should be private indeed, and distinguished from the My-Wallet-V3 _syncWallet variable.

The `syncWallet()` promise did work for me though...